src/ensembl/production/metadata/updater/core.py

```diff
@@ -36,23 +35,48 @@
 
 
 class CoreMetaUpdater(BaseMetaUpdater):
-    def __init__(self, db_uri, metadata_uri, release=None, force=None):
-        super().__init__(db_uri, metadata_uri, release, force)
+    def __init__(self, db_uri, metadata_uri, release=None):
+        super().__init__(db_uri, metadata_uri, release)
         self.db_type = 'core'
         # Single query to get all of the metadata information.
         self.meta_dict = {}
+        self._load_meta_dict()
+        self._validate_required_attributes()
+
+    def _load_meta_dict(self):
+        """Load metadata into meta_dict from the database."""
         with self.db.session_scope() as session:
-            results = session.query(Meta).all()
+            results = session.query(Meta).filter(Meta.meta_value.isnot(None),
+                                                 Meta.meta_value.notin_(['', 'Null', 'NULL'])).all()
             for result in results:
                 species_id = result.species_id
                 meta_key = result.meta_key
                 meta_value = result.meta_value
-
                 if species_id not in self.meta_dict:
                     self.meta_dict[species_id] = {}
                 # WARNING! Duplicated meta_keys for a species_id will not error out!. A datacheck is necessary for key values.
                 self.meta_dict[species_id][meta_key] = meta_value
 
+    def _validate_required_attributes(self):
```

Comment:
Are there any test cases for validating required attributes?

Reply:
It's good to add the validator as a separate module so we can add it as early step in handover.  
with this implementation, it will fail at the last stage of the handover so dc and db copy steps need to be rerun

Reply:
Honestly, it is just here until we have a proper DC. Eventually I hope to move all the necessary datachecks to python, including this. If the DC is working correctly for meta_keys, this should be superfluous.